cluster/charts/rook-ceph-cluster/values.yaml

```diff
@@ -0,0 +1,279 @@
+# Default values for a single rook-ceph cluster
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# Rook Image
+image: rook/ceph:master
+
+# Namespace of the main rook operator
+operatorNamespace: rook-ceph
+
+# Ability to override ceph.conf
+configOverride:
+#configOverride: |
+#  [global]
+#  mon_allow_pool_delete = true
+#
+#  osd_pool_default_size = 3
+#  osd_pool_default_min_size = 2
+
+# Installs a debugging toolbox deployment
+toolbox:
+  enabled: false
+  tolerations: []
+  affinity: {}
+
+monitoring:
+  # requires Prometheus to be pre-installed
+  # enabling will also create RBAC rules to allow Operator to create ServiceMonitors
+  enabled: false
+
+# All values below are taken from the CephCluster CRD
+# More information can be found at [Ceph Cluster CRD](/Documentation/ceph-cluster-crd.md)
+cephClusterSpec:
+  dataDirHostPath: /var/lib/rook
+
+  cephVersion:
+    image: ceph/ceph:v16.2.1
+    allowUnsupported: false
+  skipUpgradeChecks: false
+
+  # WaitTimeoutForHealthyOSDInMinutes defines the time (in minutes) the operator would wait before an OSD can be stopped for upgrade or restart.
+  # If the timeout exceeds and OSD is not ok to stop, then the operator would skip upgrade for the current OSD and proceed with the next one
+  # if `continueUpgradeAfterChecksEvenIfNotHealthy` is `false`. If `continueUpgradeAfterChecksEvenIfNotHealthy` is `true`, then opertor would
+  # continue with the upgrade of an OSD even if its not ok to stop after the timeout. This timeout won't be applied if `skipUpgradeChecks` is `true`.
+  # The default wait timeout is 10 minutes.
+  waitTimeoutForHealthyOSDInMinutes: 10
+
+  # The option to automatically remove OSDs that are out and are safe to destroy.
+  removeOSDsIfOutAndSafeToRemove: false
+
+  storage:
+    useAllNodes: false
```

Comment:
Let's use the same defaults as cluster.yaml, otherwise no OSDs are created by default.